0006-zigzag-conversion.py

```diff
@@ -0,0 +1,17 @@
+from math import ceil
+class Solution:
```

Comment:
If you did put a one line space between imports and 'class solution' it would be nice and distinguishable.